iceoryx_hoofs/test/moduletests/test_posix_access_control.cpp

```diff
@@ -134,7 +134,8 @@ TEST_F(AccessController_test, writeSpecialUserPermissions)
 
     auto name = iox_getpwuid(geteuid());
     ASSERT_TRUE(name);
-    PosixUser::userName_t currentUserName(iox::cxx::TruncateToCapacity, name->pwd.pw_name);
+    PosixUser::userName_t currentUserName(
+        iox::cxx::TruncateToCapacity, name->pwd.pw_name, strnlen(name->pwd.pw_name, PosixUser::userName_t::capacity()));
```

Comment:
@elfenpiff @FerdinandSpitzschnueffler Here, and in all other places where we convert `char*` to `iox::string` we rely on the operating system to supply us a properly null-terminated string.